rust/action-attribute/README.md

```
@@ -0,0 +1,6 @@
+# ephemeral-rollups-sdk-attribute-action
+
+Procedural macro attribute for committing accounts in Ephemeral Rollups.
+
+Quickstart and integration guide:
+https://docs.magicblock.gg/pages/get-started/how-integrate-your-program/quickstart
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)